src/main/webapp/dataverse.xhtml

```diff
@@ -267,6 +269,21 @@
                                     </p:inputTextarea>
                                     <p:message for="description" display="text"/>
                                 </div>
+                                </div>
+                                    <div class="col-xs-12 form-group" jsf:rendered="#{DataversePage.getGuestbookEntryOptions().size()>2}">
+                                        <h:outputLabel for="dsGuestbookEntryOption" styleClass="control-label">
+                                            #{bundle.guestbookEntryOption} 
+                                            <span class="glyphicon glyphicon-question-sign tooltip-icon"
+                                                  data-toggle="tooltip" data-placement="auto right" data-original-title="#{bundle['dataverse.guestbookentryatrequest.title']}"></span>
+                                        </h:outputLabel>
+                                        <div class="form-col-container">
+                                            <h:selectOneMenu id="dsGuestbookEntryOption" styleClass="form-control" value="#{DataversePage.dataverse.guestbookEntryAtRequest}">
```

Comment:
This is failing on update because of a property not writable exception. There is a method in the DVObjectContainer for setGuestbookEntryAtRequest but it is updating the guestbookAtRequest field that exists on the Dataverse table

Reply:
Thanks! FWIW: This was a String versus Boolean issue w.r.t. the type for guestbookEntryAtRequest - the table was originally a String like some other options (e.g. storage driver), but since it is just a flag that can be true/false/null, Boolean was simpler. I missed some places that had to change though. I think it is working now.